cg/migrations/20180618163555_update_requests.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('rectification_requests', t => {
```

Comment:
Do we really need a new table for the update requests? We could just update the subjects username, password and personal data int the existing tables
There could be a counter for how many times a subject's data was rectified, that's an interesting information for managers and the subjects themselves

Reply:
The reason for this table is so that we can implement an approval process

Reply:
I see. the approval will be automated or a manager will need to look into it?

Reply:
manager will look at it